test/tracer.js

```diff
@@ -104,10 +104,11 @@ describe('tracer should', () => {
 
         assert.equal(actualTags[0].key, 'jaeger.hostname');
         assert.equal(actualTags[1].key, 'jaeger.version');
-        assert.equal(actualTags[2].key, 'sampler.param');
-        assert.equal(actualTags[3].key, 'sampler.type');
-        assert.equal(actualTags[2].value, true);
-        assert.equal(actualTags[3].value, 'const');
+        assert.equal(actualTags[2].key, 'peer.ipv4');
+        assert.equal(actualTags[3].key, 'sampler.param');
+        assert.equal(actualTags[4].key, 'sampler.type');
+        assert.equal(actualTags[3].value, true);
+        assert.equal(actualTags[4].value, 'const');
```

Comment:
The values for these are dynamic and the helper function checks the value, meh I guess I can write a helper function that doesn't care if the values don't match but uhhhh, too lazy.